src/applications/discharge-wizard/constants/question-data-map.js

```diff
@@ -56,41 +56,40 @@ export const RESPONSES = Object.freeze({
   REASON_6: 'My discharge paperwork has another kind of error.',
   REASON_7:
     'My discharge is unjust, and it isn’t related to any of the reasons listed here.',
-  INTENTION_1: `Yes, I want to change my name, discharge date, or something written in the "other remarks" section of my DD214. (This isn't common.)`,
-  INTENTION_2:
+  INTENTION_YES: `Yes, I want to change my name, discharge date, or something written in the "other remarks" section of my DD214. (This isn't common.)`,
+  INTENTION_NO:
     'No, I want to change only my characterization of discharge, re-enlistment code, separation code, or narrative reason for discharge.',
-  COURT_MARTIAL_1:
+  COURT_MARTIAL_YES:
     'Yes, my discharge was the outcome of a general court-martial.',
-  COURT_MARTIAL_2:
+  COURT_MARTIAL_NO:
     'No, my discharge was administrative or the outcome of a special or summary court-martial.',
-  COURT_MARTIAL_3: `I'm not sure.`,
-  DISCHARGE_TYPE_1:
+  NOT_SURE: `I'm not sure.`,
+  DISCHARGE_HONORABLE:
     'My discharge is honorable or general under honorable conditions. I want to change only my narrative reason for discharge, separation code, or re-enlistment code.',
-  DISCHARGE_TYPE_2: `My discharge isn't honorable or under honorable conditions.`,
-  PREV_APPLICATION_TYPE_1:
+  DISCHARGE_DISHONORABLE: `My discharge isn't honorable or under honorable conditions.`,
+  PREV_APPLICATION_DRB_DOCUMENTARY:
     'I applied to a Discharge Review Board (DRB) for a Documentary Review.',
-  PREV_APPLICATION_TYPE_2:
+  PREV_APPLICATION_DRB_PERSONAL:
     'I applied to a Discharge Review Board (DRB) for a Personal Appearance Review.',
-  PREV_APPLICATION_TYPE_3A:
+  PREV_APPLICATION_BCMR:
     'I applied to a Board for Correction of Military Records (BCMR).',
-  PREV_APPLICATION_TYPE_3B:
+  PREV_APPLICATION_BCNR:
     'I applied to the Board for Correction of Naval Records (BCNR).',
-  PREV_APPLICATION_TYPE_4: "I'm not sure",
-  PREV_APPLICATION_1: 'Yes',
-  PREV_APPLICATION_2: 'No',
-  PREV_APPLICATION_YEAR_1A: '2014 or earlier',
-  PREV_APPLICATION_YEAR_2A: 'After 2014',
-  PREV_APPLICATION_YEAR_1B: '2011 or earlier',
-  PREV_APPLICATION_YEAR_2B: 'After 2011',
-  PREV_APPLICATION_YEAR_1C: '2017 or earlier',
-  PREV_APPLICATION_YEAR_2C: 'After 2017',
-  PRIOR_SERVICE_1:
+  YES: 'Yes',
+  NO: 'No',
+  PREV_APPLICATION_BEFORE_2014: '2014 or earlier',
+  PREV_APPLICATION_AFTER_2014: 'After 2014',
+  PREV_APPLICATION_BEFORE_2011: '2011 or earlier',
+  PREV_APPLICATION_AFTER_2011: 'After 2011',
+  PREV_APPLICATION_BEFORE_2017: '2017 or earlier',
+  PREV_APPLICATION_AFTER_2017: 'After 2017',
+  PRIOR_SERVICE_PAPERWORK:
```

Comment:
Should this be `PRIOR_SERVICE_PAPERWORK_YES` and `PRIOR_SERVICE_PAPERWORK_NO` to keep consistent with the format of other similar structures (e.g. `FAILURE_TO_EXHAUST_BCNR_YES` and `FAILURE_TO_EXHAUST_BCNR_NO`)?